source/objects/lander.lua

```diff
@@ -417,8 +417,6 @@ function Lander.draw()
 		love.graphics.draw(ship.image, x,y, math.rad(lander.angle), sx, sy, ox, oy)
 
 		-- draw flames
-		local ox = flame.width / 2
```

Comment:
Checking if these two locals are meant to be deleted?  They seem to be used 3 lines later on.

Reply:
ox = ship.width / at the top on line 406 but we still need it on line 420 because this is now flame width. But 420 doesn't need to be local.

Or even better - use different names.
shipSpriteWidth
flameSpriteWidth

It's a local variable so no danger of anything going wrong.

Reply:
naming them like shipSpriteWidth would get rid of the benefits I had in mind which are for readability and shorter lines of code. Also the description is misleading since its the half of ship width. but it's used as offset for the draw function. btw the shadowing was an intentional compromise for readability. 
But if you don't like it feel free to change it to something like shipOffsetX, shipOffsetY and flameOffsetX, flameOffsetY

Reply:
completely removing then without changing the function where ox and oy are used would result in a crash whenever the draw function is executed.

Reply:
I've reverted these 2 lines. They were shadowing, not unused.